openpos-util/src/main/java/org/jumpmind/pos/util/web/ConfiguredRestTemplate.java

```
@@ -67,18 +67,13 @@ public ConfiguredRestTemplate(int timeout) {
             @Override
             public boolean canRead(java.lang.Class<?> clazz,
                                    org.springframework.http.MediaType mediaType) {
-                return true;
+                return super.canRead(mediaType);
```

So my fix to not swallow exceptions broke XML payloads?  I suppose we could just get rid of the overrides altogether?

---

Since this message converted is added to the front of the list and was always returning true, it would try to handle any response. This change will still check media type of the message but will not check if class (or the type/contextClass below) are deserializable. Will still cause an exception at runtime instead of giving the Could not extract response. no suitable HttpMessageConverter message.